src/custom/utils/appzi.ts

```diff
@@ -42,6 +42,9 @@ declare global {
 type AppziCustomSettings = {
   userTradedOrWaitedForLong?: true
```

Comment:
So we still need to use this in addition, right? They don't allow to act as an OR with `waitedFor || expired || traded`

Reply:
I didn't try, but to simplify I rather stay with a single trigger